docs/repairs-api/intro.md

```diff
@@ -171,21 +163,14 @@ Email notification template ID is configured via [environment variables](#i-emai
 | LEASEHOLD_CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID | Gov notify email template ID for leasehold repairs, this is available once the template is created            |
 | LEASEHOLD_CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID   | Gov notify sms template ID for leasehold repairs, this is available once the template is created              |
 | LEASEHOLD_INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID     | Gov notify internal email template ID for leasehold repairs, this is available once the template is created   |
-| INTERNAL_EMAIL_PRODUCTION                       | Internal email address for receiving repair request details, for any manual follow-on process in _Production_ |
-| INTERNAL_EMAIL_STAGING                          | Internal email address for receiving repair request details, for any manual follow-on process in _Staging_    |
-| DAYS_UNTIL_IMAGE_EXPIRY_PRODUCTION              | Number in days before image uploaded by customer expires for _Production_, e.g. `14` days                     |
-| DAYS_UNTIL_IMAGE_EXPIRY_STAGING                 | Number in days before image uploaded by customer expires for _Staging_, e.g. `14` days                        |
+| INTERNAL_EMAIL                       | Internal email address for receiving repair request details, for any manual follow-on process |
```

Comment:
A few missing entries:

- CANCELLATION_INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID
- APPOINTMENT_CHANGED_SMS_NOTIFY_TEMPLATE_ID
- APPOINTMENT_CHANGED_EMAIL_NOTIFY_TEMPLATE_ID

(_May_ also be missing from the secrets page too)

Reply:
Good spot. Yes they were missing from secrets too.